### PR TITLE
[11.0][mig] mrp_progress_button

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Standard migration

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

File: mrp_progress_button/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017 Akretion (http://www.akretion.com). All Rights Reserved
+# @author Florian DA COSTA <florian.dacosta@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Mrp Progress Button",
+    "summary": "Add a button on MO to make the MO state 'In Progress'",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "website": "https://akretion.com/",
```
Comment (bodedra) [inline mrp_progress_button/__manifest__.py]: should be `https://github.com/OCA/manufacture`

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
Functionally tested. :+1: 

Attended small comments by @pedrobaeza and @bodedra

**Review by florian-dacosta — APPROVED** [review]
It seems good to me!
Thanks

**Pull request merged**